Python/Development_firmproblem/demographics.py

```diff
@@ -422,12 +416,12 @@ def get_omega(S, J, T, bin_weights, starting_age, ending_age, E, flag_graphs):
         starting_age, ending_age, S+1), poly_int_pop)
     new_omega = pop_int[1:]-pop_int[:-1]
     surv_array, children_rate = get_survival(S, starting_age, ending_age, E)
+    surv_array[-1] = 0.0
     imm_array, children_im = get_immigration2(S, starting_age, ending_age, E)
+    imm_array *= 0.0
```

Comment:
Here is the bandaid.  Eventually, this line will disappear, once we find the immigration rates bug.